tests/test_gen2.py

```diff
@@ -0,0 +1,149 @@
+# This file is part of ctrl_oods
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+import os
+import tempfile
+import unittest
+from shutil import copyfile
+import yaml
+from lsst.ctrl.oods.directoryScanner import DirectoryScanner
+from lsst.ctrl.oods.fileIngester import FileIngester
+import lsst.utils.tests
+import asynctest
+
+
+class Gen2TestCase(asynctest.TestCase):
+    """Test Scanning directory"""
+
+    def createConfig(self, config_name, fits_name, mapper):
+
+        package = lsst.utils.getPackageDir("ctrl_oods")
```

Comment:
As a general rule we shy away from using getPackageDir to find files in the test directory. We already know where those files are because we know where the test file is.

```python
testdir = os.path.abspath(os.path.dirname(__file__))
testFile = os.path.join(testdir, ...)
```

Reply:
Fixed in test_gen2.py and test_gen3.py